Library/ViewModels/DiscoveryViewModelTests.swift

```diff
@@ -85,6 +83,37 @@ internal final class DiscoveryViewModelTests: TestCase {
                                                "New params load into data source after selecting.")
   }
 
+  func testLoadRecommendedProjectsIntoDataSource_UserOptedOutOfRecommendations() {
+
+    let user = User.template
+      |> \.optedOutOfRecommendations .~ true
+
+    withEnvironment(config: Config.template, currentUser: user) {
+      self.vm.inputs.viewDidLoad()
+      self.vm.inputs.viewWillAppear(animated: false)
+
+      self.configureNavigationHeader.assertValues([initialParams])
+    }
+  }
+
+  func testLoadRecommendedProjectsIntoDataSource_UserOptedInOfRecommendations() {
```

Comment:
Somehow `OptedInOf` sounds really weird to me ... would it make more sense if we used the following?

```
_UserRecommendationsOptedOut
```
and
```
_UserRecommendationsOptedIn
```